<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1494</width>
    <height>888</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Courier New</family>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>CASPy</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../assets/logo.png</normaloff>../assets/logo.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QSplitter::handle {background: rgb(255, 255, 255)}</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Courier New</family>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tab_manager">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1494</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QMenu" name="menuCopy">
    <property name="title">
     <string>Copy</string>
    </property>
    <addaction name="actionCopy_Exact_Answer"/>
    <addaction name="actionCopy_Approximate_Answer"/>
   </widget>
   <widget class="QMenu" name="menuTab">
    <property name="title">
     <string>Tab</string>
    </property>
    <addaction name="actionNext_Tab"/>
    <addaction name="actionPrevious_Tab"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuOutput_Type">
     <property name="toolTip">
      <string>What format will the output be displayed</string>
     </property>
     <property name="title">
      <string>Output Type</string>
     </property>
     <addaction name="actionPretty"/>
     <addaction name="actionLatex"/>
     <addaction name="actionNormal"/>
    </widget>
    <addaction name="actionUnicode"/>
    <addaction name="actionLinewrap"/>
    <addaction name="menuOutput_Type"/>
    <addaction name="separator"/>
    <addaction name="actionScientific_Notation"/>
    <addaction name="actionAccuracy"/>
    <addaction name="separator"/>
    <addaction name="actionTab_List"/>
    <addaction name="separator"/>
    <addaction name="actionUseLatex"/>
    <addaction name="actionLatexFs"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuCopy"/>
   <addaction name="menuTab"/>
  </widget>
  <action name="actionCopy_Exact_Answer">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Copy Exact Answer</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
  <action name="actionPrevious_Tab">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Previous Tab</string>
   </property>
   <property name="shortcut">
    <string>Alt+Left</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionNew_Window">
   <property name="text">
    <string>New Window</string>
   </property>
  </action>
  <action name="actionUnicode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use Unicode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionNext_Tab">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Next Tab</string>
   </property>
   <property name="shortcut">
    <string>Alt+Right</string>
   </property>
  </action>
  <action name="actionLinewrap">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Linewrap</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionScientific_Notation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scientific Notation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionAccuracy">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Accuracy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionClear_Shell">
   <property name="text">
    <string>Clear Shell</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionCopy_Approximate_Answer">
   <property name="text">
    <string>Copy Approximate Answer</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionPretty">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pretty</string>
   </property>
   <property name="toolTip">
    <string>Pretty Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionLatex">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Latex</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
   <property name="toolTip">
    <string>Normal - No special format</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionExact_Answer">
   <property name="text">
    <string>Exact Answer</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
  <action name="actionApproximate_Answer">
   <property name="text">
    <string>Approximate Answer</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionClear_Thread">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Clear Thread</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionTab_List">
   <property name="text">
    <string>Tab List</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionLaTeX_resolution">
   <property name="text">
    <string>LaTeX resolution</string>
   </property>
  </action>
  <action name="actionUseLatex">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use LaTeX renderer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionLatexFs">
   <property name="text">
    <string>LaTeX font-size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
